ipaclient/install/client.py

```diff
@@ -2288,26 +2288,26 @@ def install_check(options):
 
 
 def create_ipa_nssdb():
-    db = certdb.NSSDatabase(paths.IPA_NSSDB_DIR)
-    pwdfile = os.path.join(db.secdir, 'pwdfile.txt')
+    db = certdb.NSSDatabase(paths.IPA_NSSDB_DIR,
+                            os.path.join(paths.IPA_NSSDB_DIR, 'pwdfile.txt'))
```

Comment:
I could do this change properly by replacing `NSSDatabase` with `CertDB` but that would mean having `certs.py` moved to `ipalib/install` so that it's in a reasonable spot which I am not sure is worth it.